src/main/groovy/org/pih/warehouse/core/PaginationParams.groovy

```diff
@@ -0,0 +1,23 @@
+package org.pih.warehouse.core
+
+import grails.validation.Validateable
+
+abstract class PaginationParams implements Validateable {
+
+    Integer max
+
+    Integer offset
+
+    Integer getMax() {
+        return max
+    }
+
+    Integer getOffset() {
+        return offset
+    }
```

Comment:
I think those two getters are not necessary, because Groovy/Grails adds them by default.